src/services/categories.services.ts

```diff
@@ -1,17 +1,21 @@
 import {api} from "@/boot/axios";
+import CategoryItem from "@/models/category/categories";
 
 export default new class {
-    private readonly category = 'categories';
-    getCategory(id: number){
-        return api.get(this.category + "/" + id.toString())
+    private readonly endpoint = 'categories';
+    getCategory(id: string){
+        return api.get(`${this.endpoint}${"/"}${id}`)
     }
     getCategories(){
-        return api.get(this.category);
+        return api.get(this.endpoint);
     }
-    getBookCategory(categoryId: number){
-        return api.get(this.category + "/" + categoryId.toString() + "/books")
+    getBookCategory(categoryId: string){
+        return api.get(`${this.endpoint}/${categoryId}${"/books"}`)
```

Comment:
Chỉ dùng cú pháp `${}` cho biến và biểu thức thôi, cái string row thì không cần
```suggestion
        return api.get(`${this.endpoint}/${categoryId}/books`)
```

Reply:
done